### PR TITLE
Added hover effect to footer external links

### LINKED ISSUE #484: Sub-Footer Section Hovering
### Would you like to have a new feature? Please describe.

I Feel it would really look amazing in the perspective OF User Interface when people will come onto the Sub-Footer section which is containing Organisation, Community, Socials & Contact , to each of the options available under them, if we make hoverable edits like color changes to   #5942e9 and text written gets little scaled up...I've tried out the feature and looks really cool!

Same thing I'm thinking for Social Buttons too!
Will be grateful, if I Can contribute too!
Please Assign!

### How would this feature be beneficial for the project

https://user-images.githubusercontent.com/68687087/224511389-335601f2-dacb-40e7-b810-1f6d883040b1.mp4

See, How Cool it is resembling!

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Varunvaru524**
## Fixes Issue

This PR fixes the following issues:

closes #484 

## Changes proposed

Added the navbar hover animation to the footer external links so that the links will be more visible and the style of the project will stay consistent throughout.

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

https://user-images.githubusercontent.com/101904494/224563944-8940545a-b8fe-4f3d-b0e8-67cee3b1e368.mp4

### CODE REVIEW

**Comment by Chandraprakash-Darji** [comment]
@Varunvaru524  great work

**Comment by Chandraprakash-Darji** [comment]
Fix the conflicts

**Review by Varunvaru524 — COMMENTED** [review]
Fixed all the conflicts

File: README.md [hunk 1]
```diff
@@ -11,7 +11,6 @@
 <a href="https://github.com/wemakedevs/wemakedevs/blob/main/LICENSE"><img alt="GitHub license" src="https://img.shields.io/github/license/wemakedevs/wemakedevs"></a>
 </p>
 
-
```
Comment (Chandraprakash-Darji) [inline README.md:14]: undo update in the readme

File: src/components/layout/Footer/index.module.css [hunk 2]
```diff
@@ -98,6 +98,10 @@
   font-weight: 400;
 }
 
+.linkGroups li a {
+  padding-bottom: 3px;
```
Comment (Chandraprakash-Darji) [inline src/components/layout/Footer/index.module.css:102]: share the preview without this

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

File: src/components/layout/Footer/index.module.css (hunk 2 above)
Comment (Varunvaru524) [reply src/components/layout/Footer/index.module.css:102]: Without 3px Padding

https://user-images.githubusercontent.com/101904494/226563421-8ff62c03-3a1d-4d4f-8074-c785a8010c13.mp4

File: README.md (hunk 1 above)
Comment (Varunvaru524) [reply README.md]: Undid Readme file
[8ec396b](https://github.com/WeMakeDevs/wemakedevs/pull/485/commits/8ec396bcc71352ae2cfc95c4e040bcdd632fddec)

**Review by Chandraprakash-Darji — APPROVED** [review]

**Comment by Varunvaru524** [comment]
Is this PR ready to merge?

**Pull request merged**